package.json

```
@@ -128,5 +140,29 @@
 	"engines": {
 		"node": ">=14",
 		"npm": ">=6"
+	},
+	"husky": {
+		"hooks": {
+			"commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+			"pre-commit": "yarn lint-staged",
+			"post-commit": "git update-index --again"
+		}
+	},
+	"commitlint": {
+		"extends": [
+			"@commitlint/config-conventional"
+		]
+	},
+	"lint-staged": {
+		"*.ts": [
```

```suggestion
		"*.{mjs,js,ts}": [
```
Should run eslint on .mjs, .js and .ts files